pywr/_recorders.pxd

```diff
@@ -50,6 +50,15 @@ cdef class NumpyArrayParameterRecorder(ParameterRecorder):
 cdef class NumpyArrayIndexParameterRecorder(IndexParameterRecorder):
     cdef int[:, :] _data
 
+cdef class FlowDurationCurveRecorder(NumpyArrayNodeRecorder):
+    cdef double[:] _percentiles
+    cdef double[:, :] _fdc_flows
```

Comment:
`_fdc_fows`. I think the name could be better. Either just `fdc` or `flow_duration_curve`